src/Processors/Transforms/WindowTransform.cpp

```diff
@@ -2252,6 +2349,13 @@ void registerWindowFunctions(AggregateFunctionFactory & factory)
             return std::make_shared<WindowFunctionExponentialTimeDecayedAvg>(
                 name, argument_types, parameters);
         }, properties});
+
+    factory.registerFunction("nonNegativeDerivative", {[](const std::string & name,
```

Comment:
Could/should this method also be documented in the official documentation?

Reply:
Yup, I will add it.

Reply:
I will add it in a separate PR